mods/common/scripts/utils.lua

```diff
@@ -12,3 +12,15 @@ IdleHunt = function(actor)
 		Trigger.OnIdle(actor, actor.Hunt)
 	end
 end
+
+AddPrimaryObjective = function(player, description)
```

Comment:
Does the Lua extension we use have support for method documentation comments so we can explain that these wrappers' point is to provide translation?

Reply:
Yes, good idea: added https://emmylua.github.io/annotations/